src/javascript/app_2/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -74,3 +74,34 @@ const createProposalRequestForContract = (store, type_of_contract) => {
         ),
     };
 };
+
+export const getProposalParametersName = (proposal_requests, alternatives, removable_properties) => {
```

Comment:
I believe it's possible to simplify the whole function. Let's check this together.

Reply:
@binary-hamedanchi 
Sure let talk about it whenever you're free.